client/src/store/store.ts

```diff
@@ -10,6 +10,16 @@ import loginReducer from "./slices/login";
 import configReducer from "./slices/config";
 import mentorReducer from "./slices/mentor";
 import questionsReducer from "./slices/questions";
+import * as Sentry from "@sentry/react";
+
+const sentryEnhancer = Sentry.createReduxEnhancer({
+  actionTransformer: (action) => {
+    if (action.error) {
+      Sentry.captureException(action.error.message);
```

Comment:
think you're supposed to pass in an actual Error instance, so probably `action.error`